src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md

```diff
@@ -182,6 +182,7 @@ The following table describes the instructions specific for page configuration f
           <li><code>&lt;css&gt;</code></li>
           <li><code>&lt;font&gt;</code></li>
           <li><code>&lt;script&gt;</code></li>
+          <li><code>&lt;remove&gt;</code></li>
```

Comment:
Hi @abrarpathan19. Thank you for your collaboration. Could you add the `attribute` element as well, please? So we have the complete list of elements. Refer to the following files for the details

https://github.com/magento/magento2/blob/2.3/lib/internal/Magento/Framework/View/Layout/etc/head.xsd#L69
https://github.com/magento/magento2/blob/2.3/app/code/Magento/Catalog/view/frontend/layout/catalog_product_opengraph.xml#L10

Thank you!

Reply:
@rogyar, thanks for the suggestions but I am not able to understand your concerns. Please explain in detail, again please

Reply:
@abrarpathan19  If you check the schema file that @rogyar linked, you can see the elements that are supported within the <head> section:

```    <xs:complexType name="headType">
        <xs:sequence minOccurs="0" maxOccurs="unbounded">
            <xs:element name="title" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="css" type="linkType" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="link" type="linkType" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="meta" type="metaType" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="script" type="scriptType" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="remove" type="srcRemoveType" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="attribute" type="headAttributeType" minOccurs="0" maxOccurs="unbounded"/>
            <xs:element name="font" type="linkType" minOccurs="0" maxOccurs="unbounded"/>
        </xs:sequence>
    </xs:complexType>
```
In the current table, the `<remove>` and `<attribute>` elements are missing.

![image](https://user-images.githubusercontent.com/6391769/78468641-6d320280-76df-11ea-9dcf-80e5e2afd19f.png)

You added the `<remove>` element. You can also add the `<attribute>` element

Reply:
@rogyar  @meker12 
I have added requested code. 

Thanks for the suggestions.